lib/ddtrace/utils/mass_tagger.rb

```diff
@@ -0,0 +1,59 @@
+module Datadog
+  module Utils
+    # Helper class to used to tag configured headers
+    module MassTagger
```

Comment:
Could you add unit tests for this module?

Reply:
I renamed this to Tagger. And it now has tests  👍